packages/cli/src/imperative.ts

```diff
@@ -236,7 +236,7 @@ const config: IImperativeConfig = {
                         "specified on every command"
                 },
                 {
-                    options: "zosAPIML --host zosAPIML --port 2020 --user ibmuser --password myp4ss --reject-unauthorized false --base-path basePath",
+                    options: "zosAPIML --host zosAPIML --port 60004 --user ibmuser --password myp4ss --reject-unauthorized false --base-path ibmzosmf/api/v1",
```

Comment:
Please change the port in the example to 7554. The engineering team has determined that it's a more common default value.

The same change should also be made in the readme file.